test/operations.jl

```diff
@@ -15,9 +16,11 @@ end
     @test reg1!=reg2
     @test statevec(reg2) == onehot(ComplexF64, nbit, 4)
     reg3 = reg1 + reg2
+
     @test statevec(reg3) == onehot(ComplexF64, nbit, 4) + onehot(ComplexF64, nbit, 0)
     @test statevec(reg3 |> normalize!) == (onehot(ComplexF64, nbit, 4) + onehot(ComplexF64, nbit, 0))/sqrt(2)
     @test (reg1 + reg2 - reg1) == reg2
+    @test reg1' + reg2' - reg1' == reg2'
```

Comment:
please also add tests for `Number * reg'`, `reg' * number`, `isnormalized(reg')`, `normalize!(reg')`, `-reg'`